input/fsh/SD_Group.fsh

```diff
@@ -0,0 +1,31 @@
+Profile: MCodePatientGroup
+Parent: Group
+Id: mcode-patient-group
+Title: "mCODE Patient Group"
+Description: "Identifies all mCODE patients for a given mCODE Data Sender.
+
+Conformance Statement:
+
+This group MUST contain all Patient resources considered by the implementer to be mCODE Cancer
+Patients. These Patient resources therefore MUST conform to [CancerPatient].
+
+[CancerPatient]: StructureDefinition-mcode-cancer-patient.html
+"
+
+* type = #person
+* actual = true
+* code 0..0 // Per https://www.hl7.org/fhir/group-definitions.html#Group.code, should be omitted for Person resources.
+
+// "Identifies traits whose presence or absence is shared by members of the group."
+// https://www.hl7.org/fhir/group-definitions.html#Group.characteristic.code
+* characteristic 1..*
+* characteristic.code = #administrative
+* characteristic.valueCodeableConcept = #mcode-cancer-patient "mCODE Cancer Patient"
```

Comment:
Specifying `characteristic.valueCodeableConcept` is the only way I could see to cleanly identify a `Group` resource as conforming to this profile.

I don't have a strong opinion about the `valueCodeableConcept` value that's used here. If there's a better option than what I have, please let me know.